libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -38,20 +38,18 @@
           }}
         </option>
       </select>
-    </sky-input-box>
-    <sky-form-errors [labelText]="labelText">
       <sky-form-error
         *ngIf="
-          hostControl?.errors?.['skyDateRange']?.errors.endDateBeforeStartDate
+          hostControl?.errors?.['skyDateRange']?.errors
+            .endDateBeforeStartDate && labelText
```

Comment:
date range picker had `sky-form-errors` outside of the dropdown input box to have the error span the entire length of date range picker. We have moved it into the first input box and the error will now show under the first box in date ranger picker. This is to keep it in alignment with all the other errors.